scipy/interpolate/_aaa.py

```diff
@@ -309,6 +319,72 @@ def _compute_weights(self, f, z, rtol, max_terms):
         self.support_values = fj[i_non_zero]
         self.weights = wj[i_non_zero]
         self.errors = errors[: m + 1]
+        self._points = z
+        self._values = f
+
+    def clean_up(self, cleanup_tol=1e-13):
```

Comment:
I have included `clean_up` as a public method in case the user needs to clean up again with a different tolerance so they don't have to recompute the whole approximation

Reply:
Every instance of `cleanup` or `clean_up` makes me progressively less of a fan of the name.